recipes/corpus/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "Corpus" %}
+{% set version = "0.4.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ffa894899350675f874bab9e93ef07326aeb5fc1e9ef62d25729d4b2903c63cc
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - Corpus
+  requires:
+    - nose
+
+about:
+  home: https://github.com/aneeshb413/Corpus
+  license: MIT
+  license_family: MIT
+  summary: A simple English dictionary for Python
+  doc_url: 
+  dev_url: 
+
+extra:
+  recipe-maintainers:
+    - talhajunaidd
+    - sadeemahmad
```

Comment:
@sadeemahmad, are you OK being added as maintainer for this conda package?

Reply:
Yes